An exception occurs on attempt to use change context step

**Describe the bug**
An exception occurs on attempt to use change context step

**To Reproduce**
1 Add following plugins to build.gradle:
    implementation(group: 'org.vividus', name: 'vividus', version: vividusVersion)
    implementation(group: 'org.vividus', name: 'vividus-plugin-web-app', version: vividusVersion)
    implementation(group: 'org.vividus', name: 'vividus-plugin-rest-api', version: vividusVersion)
    implementation(group: 'org.vividus', name: 'vividus-plugin-visual', version: vividusVersion)
    implementation(group: 'org.vividus', name: 'vividus-plugin-datetime', version: vividusVersion)
    implementation(group: 'org.springframework', name: 'spring-beans', version: '5.2.5.RELEASE')
2 Create new *.story file with step:
 When I change context to element located `By.xpath(....)`
3 Run story using command with required params:
 sh vividus-starter-0.1.0-SNAPSHOT/scripts/testRunner.sh ...

**Expected behavior**
COntext is changed successfully

**Artifacts**
(org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'org.vividus.bdd.steps.ui.GenericSetContextSteps' available: expected single matching bean but found 2: genericSetContextSteps,setContextSteps)
2020-11-16 13:41:48,839 [batch-1-thread-1] ERROR org.vividus.bdd.log.LoggingStoryReporter - Step is failed with exception
org.jbehave.core.failures.UUIDExceptionWrapper: When I change context to element located `By.xpath(.//div[@class='slds-template__container'])`
        at org.jbehave.core.steps.StepCreator$ParametrisedStep.perform(StepCreator.java:856) ~[jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$FineSoFar.run(PerformableTree.java:390) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$FineSoFar.run(PerformableTree.java:397) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$FineSoFar.run(PerformableTree.java:397) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$FineSoFar.run(PerformableTree.java:397) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$PerformableSteps.perform(PerformableTree.java:1334) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$AbstractPerformableScenario.perform(PerformableTree.java:1211) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$AbstractPerformableScenario.performRestartableSteps(PerformableTree.java:1190) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$AbstractPerformableScenario.performScenario(PerformableTree.java:1202) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$ExamplePerformableScenario.perform(PerformableTree.java:1273) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$PerformableScenario.perform(PerformableTree.java:1100) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$PerformableStory.performScenarios(PerformableTree.java:992) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree$PerformableStory.perform(PerformableTree.java:960) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree.performCancellable(PerformableTree.java:469) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.PerformableTree.perform(PerformableTree.java:436) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.StoryManager$EnqueuedStory.call(StoryManager.java:297) [jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.embedder.StoryManager$EnqueuedStory.call(StoryManager.java:270) [jbehave-core-4.8.0-alpha.6.jar:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:830) [?:?]
Caused by: org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'org.vividus.bdd.steps.ui.GenericSetContextSteps' available: expected single matching bean but found 2: genericSetContextSteps,setContextSteps
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveNamedBean(DefaultListableBeanFactory.java:1260) ~[spring-beans-5.3.0.jar:5.3.0]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveBean(DefaultListableBeanFactory.java:494) ~[spring-beans-5.3.0.jar:5.3.0]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:349) ~[spring-beans-5.3.0.jar:5.3.0]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:342) ~[spring-beans-5.3.0.jar:5.3.0]
        at org.vividus.bdd.steps.SpringStepFactory.createInstanceOfType(SpringStepFactory.java:39) ~[vividus-bdd-engine-0.2.10-SNAPSHOT.jar:0.2.10-SNAPSHOT]
        at org.jbehave.core.steps.StepCreator.stepsInstance(StepCreator.java:100) ~[jbehave-core-4.8.0-alpha.6.jar:?]
        at org.jbehave.core.steps.StepCreator$ParametrisedStep.perform(StepCreator.java:834) ~[jbehave-core-4.8.0-alpha.6.jar:?]


**Execution environment (please complete the following information):**
 - Vividus version: 0.2.10
 - OS: Window/Linux

--- a/vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java
+++ b/vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java
@@ -30,7 +30,6 @@ import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.vividus.bdd.monitor.TakeScreenshotOnFailure;
 import org.vividus.bdd.steps.StringComparisonRule;
-import org.vividus.bdd.steps.ui.GenericSetContextSteps;
 import org.vividus.bdd.steps.ui.validation.IBaseValidations;
 import org.vividus.bdd.steps.ui.validation.IDescriptiveSoftAssert;
 import org.vividus.selenium.IWebDriverProvider;
@@ -44,23 +43,27 @@ import org.vividus.ui.web.action.search.WebLocatorType;
 import org.vividus.ui.web.util.LocatorUtil;
 
 @TakeScreenshotOnFailure
-public class SetContextSteps extends GenericSetContextSteps
+public class SetContextSteps
 {
     private static final String AN_ELEMENT_WITH_THE_ATTRIBUTE = "An element with the attribute '%1$s'='%2$s'";
 
+    private final IUiContext uiContext;
     private final IWebDriverProvider webDriverProvider;
     private final IDescriptiveSoftAssert descriptiveSoftAssert;
     private final IWindowsActions windowsActions;
     private final IWaitActions waitActions;
+    private final IBaseValidations baseValidations;
 
-    public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvider, IBaseValidations baseValidations,
-            IDescriptiveSoftAssert descriptiveSoftAssert, IWindowsActions windowsActions, IWaitActions waitActions)
+    public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvider,
+            IDescriptiveSoftAssert descriptiveSoftAssert, IWindowsActions windowsActions, IWaitActions waitActions,
+            IBaseValidations baseValidations)
     {
-        super(uiContext, baseValidations);
+        this.uiContext = uiContext;
         this.webDriverProvider = webDriverProvider;
         this.descriptiveSoftAssert = descriptiveSoftAssert;
         this.windowsActions = windowsActions;
         this.waitActions = waitActions;
+        this.baseValidations = baseValidations;
     }
 
     /**
@@ -74,11 +77,11 @@ public class SetContextSteps extends GenericSetContextSteps
     public void changeContextToElementWithName(State state, String name)
     {
         resetContext();
-        WebElement element = getBaseValidations().assertIfElementExists(
+        WebElement element = baseValidations.assertIfElementExists(
                 String.format("An element with the name '%1$s'", name),
                 new Locator(WebLocatorType.ELEMENT_NAME, name)
                         .addFilter(WebLocatorType.STATE, state.toString()));
-        getUiContext().putSearchContext(element, () -> changeContextToElementWithName(state, name));
+        uiContext.putSearchContext(element, () -> changeContextToElementWithName(state, name));
     }
 
     /**
@@ -92,11 +95,11 @@ public class SetContextSteps extends GenericSetContextSteps
     {
         resetContext();
 
-        WebElement element = getBaseValidations().assertIfElementExists(
+        WebElement element = baseValidations.assertIfElementExists(
                 String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE, attributeType, attributeValue),
                 new Locator(WebLocatorType.XPATH,
                         LocatorUtil.getXPathByAttribute(attributeType, attributeValue)));
-        getUiContext().putSearchContext(element, () -> changeContextToElementWithAttribute(attributeType,
+        uiContext.putSearchContext(element, () -> changeContextToElementWithAttribute(attributeType,
                 attributeValue));
     }
 
@@ -112,14 +115,14 @@ public class SetContextSteps extends GenericSetContextSteps
     public void changeContextToStateElementWithAttribute(State state, String attributeType, String attributeValue)
     {
         resetContext();
-        WebElement element = getBaseValidations().assertIfElementExists(String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE,
+        WebElement element = baseValidations.assertIfElementExists(String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE,
                 attributeType, attributeValue), new Locator(WebLocatorType.XPATH,
                         LocatorUtil.getXPathByAttribute(attributeType, attributeValue)));
-        if (!getBaseValidations().assertElementState("The found element is " + state, state, element))
+        if (!baseValidations.assertElementState("The found element is " + state, state, element))
         {
             element = null;
         }
-        getUiContext().putSearchContext(element, () -> changeContextToStateElementWithAttribute(state,
+        uiContext.putSearchContext(element, () -> changeContextToStateElementWithAttribute(state,
                 attributeType, attributeValue));
     }
 
@@ -165,7 +168,7 @@ public class SetContextSteps extends GenericSetContextSteps
     public void switchingToFrame(Locator locator)
     {
         resetContext();
-        WebElement element = getBaseValidations().assertIfElementExists("A frame", locator);
+        WebElement element = baseValidations.assertIfElementExists("A frame", locator);
         switchToFrame(element);
     }
 
@@ -291,5 +294,10 @@ public class SetContextSteps extends GenericSetContextSteps
     private WebDriver getWebDriver()
     {
         return webDriverProvider.get();
+    }
+
+    private void resetContext()
+    {
+        uiContext.reset();
     }
 }

